src/analyze/analyze-architectures.c

```diff
@@ -1,7 +1,7 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
-#include "analyze.h"
 #include "analyze-architectures.h"
+#include "analyze.h"
```

Comment:
Can we invert this sorting so that `-` sorts after `.`? In general it'd look much better.

Reply:
No alphabetical is alphabetical unfortunately

Reply:
If I read the documentation https://clang.llvm.org/docs/ClangFormatStyleOptions.html#includecategories correctly, the headers are sorted by the first regexpr group specified in `IncludeCategories.Regex`. Therefore, adding something like:
```
IncludeCategories:
  - Regex: '"(.*)\.h"'
    Priority 999
```
should do the trick, because the value sorted does not include the `.h` extension and `analyze` sorts before `analyze-architecture`. This is untested, of course 🙂

Reply:
Does not work

Reply:
@keszybz I've filed https://github.com/llvm/llvm-project/pull/137840 now and regenerated the commit sorting patch with that PR applied to get the desired result.